### ISSUE TITLE
Usage of annotate_models gem results in a false positive RuboCop::Cop::Style::Documentation violation

### ISSUE THREAD

**Issue by topher6345** [bbatsov/rubocop#2097]
```ruby
# == Schema Information
#
# Table name: users
#
#  id                        :integer          not null, primary key
#  email                     :string(255)      default("")
#  encrypted_password        :string(255)      default(""), not null
#  reset_password_token      :string(255)
#  reset_password_sent_at    :datetime
#  remember_created_at       :datetime
#  sign_in_count             :integer          default(0), not null
#  current_sign_in_at        :datetime
#  last_sign_in_at           :datetime
#  current_sign_in_ip        :string(255)
#  last_sign_in_ip           :string(255)
#  created_at                :datetime
#  updated_at                :datetime
#
class User < ActiveRecord::Base
...
end
```

will trigger a `RuboCop::Cop::Style::Documentation` violation.

In a Rails project, this means I have to exclude this rule for every ActiveRecord model.

**Comment by jonas054** [comment]
I could not reproduce the problem. Are you sure it looks exactly like that?

**Comment by topher6345** [comment]
I apologize for having to sanitize the file.

Here is an unsanitized file - 

```ruby
# == Schema Information
#
# Table name: scores
#
#  id           :integer          not null, primary key
#  user_id      :integer
#  day_points   :integer          default(0)
#  week_points  :integer          default(0)
#  total_points :integer          default(0)
#  created_at   :datetime
#  updated_at   :datetime
#

class Score < ActiveRecord::Base
  belongs_to :user

  validates :user, presence: true
end
```

```
app/models/score.rb:14:1: C: Missing top-level class documentation comment.
class Score < ActiveRecord::Base
^^^^^
```

**Comment by jonas054** [comment]
Good. Then we know the reason. Parser associates the comment with the class only if there's no blank line between them. This seems reasonable to me, and the behavior of this gem, to put that blank line there, more questionable.

**Comment by bbatsov** [comment]
Indeed. You can ping @whitequark about this, but as far as we're concerned there's nothing for us to do about this ticket.

**Comment by bbatsov** [comment]
(except maybe adding a cop checking for empty lines between class/modules and comments preceding them)
